reduction/lr_reduction/event_reduction.py

```diff
@@ -381,6 +382,8 @@ def __init__(
         self.summing_threshold = None
         self.q_summing = False
         self.dq_over_q = 0
+        self.wavelength = []
```

Comment:
we don't need attributes `wavelength` or `d_lamba` because the output of `compute_wavelength_resolution()` will eventually make it as a factor in the final `self.dq_over_q`.